src/framework/ui/view/iconcodes.h

```diff
@@ -384,6 +384,7 @@ class IconCode
 
         PLUGIN = 0xF440,
         LYRICS = 0xF441,
+        LEARN = 0xF441,
```

Comment:
It's the same as the previous one. Is that intentional?

Reply:
Yes, that's intentional, change suggested by @igorkorsukov

Reply:
> It's the same as the previous one. Is that intentional?

Used for different purposes, but now has the same value (icon)

Reply:
But if we want to change the  image of the “learn” icon, we will have to enter a new one and look for all the places in the source code where it is used